bootstrap.py

```diff
@@ -57,12 +57,15 @@ def download_from_s3(source, destination):
 
 def instance_metadata(item):
```

Comment:
This creates an inconsistent return from `instance_metadata()`, making anything that calls it more brittle (in that the caller now needs to be aware of the callee's implementation).

The `try` here is actually reasonable though. :+1: